### PR TITLE
user/tailspin: new package

### PR DESCRIPTION
**Pull request by sonata-chen**

### CODE REVIEW

File: user/tailspin/template.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+make_check_args = ["--bins"]  # disable integration tests
+hostmakedepends = ["cargo-auditable"]
+makedepends = ["rust-std"]
+pkgdesc = "Log file highlighter"
+maintainer = "sonata-chen <sonatachen39@gmail.com>"
+license = "MIT"
+url = "https://github.com/bensadeh/tailspin"
+source = [
+    f"{url}/archive/refs/tags/{pkgver}.tar.gz",
+]
```
Comment (nekopsykose) [inline user/tailspin/template.py:14]: don't need the [] here

**Comment by triallax** [comment]
any particular reason why this is in user instead of contrib?

**Comment by sonata-chen** [comment]
> any particular reason why this is in user instead of contrib?
no

**Comment by q66** [comment]
honestly i think for random shit that's to be used by 1 person we should use user more

**Pull request merged**